docs/platform/howto/move-to-aws-marketplace-billing.rst

```diff
@@ -0,0 +1,28 @@
+Move from Aiven direct billing to AWS Marketplace
+=================================================
+
+Aiven makes its services available through the AWS Marketplace.  If you already have some services running in a project which is billed directly through Aiven but you would like to move to an AWS Marketplace subscription without disrupting your services, this article shows the steps needed to gather the relevant information and submit the request.
+
+Create a new account using an AWS Marketplace subscription
+----------------------------------------------------------
+
+Follow the steps to :doc:`set up AWS Marketplace for Aiven Services <billing-aws-marketplace-subscription>`.  This will create a new Aiven organization, which is where your project will be moved to.
```

Comment:
```suggestion
Follow the steps to :doc:`set up AWS Marketplace for Aiven services <billing-aws-marketplace-subscription>`.  This will create a new Aiven organization, which is where your projects will be moved to.
```